snapshot/overlay/overlay.go

```diff
@@ -114,7 +114,18 @@ func (o *Snapshotter) Prepare(key, parent string) ([]containerd.Mount, error) {
 }
 
 func (o *Snapshotter) View(key, parent string) ([]containerd.Mount, error) {
-	panic("not implemented")
+	if parent == "" {
```

Comment:
Do we not allow viewing an empty directory?

We should define that boundary condition.